site/content/integrate/apps/structure/forms/_index.md

```diff
@@ -0,0 +1,712 @@
+| {{<newtabref title="email" href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/email">}}       | A field for editing an email address.                            |
+| {{<newtabref title="number" href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/number">}}     | A field for entering a number; includes a spinner component.     |
+| {{<newtabref title="password" href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/password">}} | A single-line text input field whose value is obscured.          |
+| {{<newtabref title="tel" href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/tel">}}           | A field for entering a telephone number.                         |
+| {{<newtabref title="url" href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/url">}}           | A field for entering a URL.                                      |
+
+#### Select fields
+
+##### Select options
+
+The data structure of an option (`SelectOption`) in a select field is defined by the following table:
+
+| Name                                                  | Type   | Description                                                                                     |
+|:------------------------------------------------------|:-------|:------------------------------------------------------------------------------------------------|
+| `label`{{<compass-icon icon-star "Mandatory Value">}} | string | User-facing string. Defaults to `value` and must be unique on this field.                       |
+| `value`{{<compass-icon icon-star "Mandatory Value">}} | string | Machine-facing value. Must be unique on this field.                                             |
+| `icon_data`                                           | string | Either a fully-qualified URL, or a path for an app's [static asset]({{<ref "static-assets">}}). |
+
+##### Dynamic select options
+
+A dynamic select field gets its options by performing the `lookup` call. The [call request]({{<ref "call#request">}}) will include the App, user, channel, and team IDs in the context.
+The [call response]({{<ref "call#response">}}) is expected to contain the list of select field options in the `items` key of the `data` map. Each option is of the type [SelectOption](#select-options).
+
+Example `lookup` call response:
+
+```json
+{
+    "type": "ok",
+    "data": {
+        "items": [
+            {
+                "label": "Option One",
+                "value": "option_1"
+            },
+            {
+                "label": "Option Two",
+                "value": "option_2"
+            }
+        ]
+    }
+}
+```
+
+##### Select field refresh
+
+If the `refresh` value is set to `true`, the form's `source` call is performed any time the field's value changes. The call request will include the App, user, channel, and team IDs in the context; the current values of the form are also included.
```

Comment:
> The call request will include the App, user, channel, and team IDs in the context

Are these values sent to the App, without them explicitly being expanded?

Reply:
I think this text, which appears a few times in the Forms topic, is incorrect since the `expand` functionality was added. I've removed the last sentence entirely from the above and have done similarly for the other Forms examples.